hack/ci-release.sh

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+# ci-release.sh - automated a release of Decco for CI systems. The version to
+#                 release is set in the VERSION file.
+#
+# The script assumes a Linux host with Docker and Make installed. It
+# installs all further prerequisites (like Go, goreleaser) and does the
+# git tagging. If you want to create a release without all this, you can use
+# `make release`.
+#
+# Expected environment variables:
+#
+# - DOCKER_PASSWORD       The password for the Docker account to push the images too.
+# - DOCKER_USERNAME       The username for the Docker account to push the images too.
+# - DOCKER_REGISTRY       The Docker registry to push the images too.
```

Comment:
Where is this used?

Reply:
You are right, is not being used. Will remove it 👍